src/metabase/api/card.clj

```diff
@@ -1001,14 +1001,14 @@ saved later when it is ready."
         schema-name       (setting/get :uploads-schema-name)
         filename-prefix   (or (second (re-matches #"(.*)\.csv$" filename))
                               filename)
-        table-name        (-> (str (setting/get :uploads-table-prefix) filename-prefix)
-                              upload/unique-table-name)
-        schema+table-name (if (str/blank? schema-name)
-                            table-name
-                            (str schema-name "." table-name))
         driver            (driver.u/database->driver database)
         _                 (or (driver/database-supports? driver :uploads nil)
                               (throw (Exception. (tru "Uploads are not supported on {0} databases." (str/capitalize (name driver))))))
+        table-name        (->> (str (setting/get :uploads-table-prefix) filename-prefix)
```

Comment:
why the use of `setting/get` instead of `(uploads-table-prefix)` as a function call?

Reply:
Good point, `setting/get` was leftover from something we used to have. Updated https://github.com/metabase/metabase/commit/ac66fddb91d190161695c1ddef76d56eb706cec7